src/jit/jitconfigvalues.h

```diff
@@ -203,6 +203,12 @@ CONFIG_STRING(JitNoInlineRange, W("JitNoInlineRange"))
 CONFIG_STRING(JitInlineReplayFile, W("JitInlineReplayFile"))
 #endif // defined(DEBUG) || defined(INLINE_DATA)
 
+#ifdef FEATURE_CORECLR
+CONFIG_INTEGER(JitInlinePolicyLegacy, W("JitInlinePolicyLegacy"), 0)
+#else
+CONFIG_INTEGER(JitInlinePolicyLegacy, W("JitInlinePolicyLegacy"), 1)
+#endif
+
```

Comment:
Please enable the new policy by default even for non-CoreCLR. We will opt out later if we determine we don't want this new behavior for desktop.